src/lib/pay-request/api_utils/ledger.js

```diff
@@ -47,7 +47,25 @@ const ledgerMethods = function ledgerMethods(instance) {
       .then(utilExtractData)
   }
 
-  return { transaction, transactions, events }
+  const statistics = function statistics(account, fromDate, toDate, override) {
```

Comment:
I think it would be a nice abstraction for the controller not to worry about the concept of 'Overriding', that's an API specific question and so the REST layer (here) would be a great place to deal with it. 

Maybe something like 
```js
if (account) { 
  params.account_id = account
} else { 
  params.override_account_id = true
}
```

The interface here would then say if you provide an account it will return data for that account, if not it will be a platform wide query.

Reply:
Done